### PR TITLE
Benchmark batch size vs row size

### PR DESCRIPTION
**Pull request by shuhaowu**
Some scripts to benchmark Ghostferry batch size vs row size.

With this here, others can try it out too and possibly modify it for their own uses.

### CODE REVIEW

File: benchmark/studies/batch_size.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+require "fileutils"
+
+require_relative "../ghostferry_benchmark.rb"
+
+row_sizes = [1, 100, 200, 500]
+batch_sizes = (5000..10000).step(1000).to_a
+
+max_total_bytes = 100_000_000
```
Comment (Manan007224) [inline benchmark/studies/batch_size.rb:8]: Is this the max packet size that can be written to mysql ?

**Review by Manan007224 — APPROVED** [review]

File: benchmark/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Ghostferry benchmark setup
```
Comment (Manan007224) [inline benchmark/README.md:1]: Ia this benchmark setup specific to linux environment, and if yes we should mention it in the README

**Comment by Manan007224** [comment]
LGTM. Although I haven't reviewed the python analysis in depth, as am unfamiliar with how these python libraries work.

File: benchmark/studies/batch_size.rb (hunk 1 above)
Comment (shuhaowu) [reply benchmark/studies/batch_size.rb:8]: No. It's just a maximum byte size I want to insert to the source so the benchmark doesn't run forever.

**Pull request merged**